tests/phpunit/Admin/SettingsTest.php

```
@@ -0,0 +1,223 @@
+		}
+	}
+
+	public function test_get_scannable_post_statuses() {
+		$statuses = Settings::get_scannable_post_statuses();
+		$this->assertEquals( [ 'publish', 'future', 'draft', 'pending', 'private' ], $statuses );
+	}
+
+	public function test_get_scannable_post_types_free_version_default() {
+		self::$mock_edac_post_types = [ 'post', 'page' ];
+		self::$mock_valid_post_types = [ 'post' => 'post', 'page' => 'page', 'attachment' => 'attachment' ];
+		$post_types = Settings::get_scannable_post_types();
```

PHPMD (2.15.0)

Avoid using static access to class '\EDAC\Admin\Settings' in method 'test_get_scannable_post_types_free_version_default'. (Clean Code Rules)

(StaticAccess)